pkg/backend/storage/fs/store.go

```diff
@@ -54,7 +55,25 @@ func (s *Store) Set(ctx context.Context, name string, value []byte) error {
 func (s *Store) Delete(ctx context.Context, name string) error {
 	path := filepath.Join(s.path, filepath.Clean(name))
 	out.Debug(ctx, "fs.Delete(%s) - %s", name, path)
-	return os.Remove(path)
+
+	err := os.Remove(path)
+	if err != nil {
```

Comment:
I would normally put the err assignment one the same line as the if.

Reply:
yes, that looks much better :)